product_template_tags/models/product_template_tag.py

```diff
@@ -28,5 +28,12 @@ def _default_company(self):
     @api.multi
     @api.depends('product_tmpl_ids')
     def _compute_products_count(self):
+        if not self.ids:
+            return
+        self.env.cr.execute("""SELECT tag_id, COUNT(DISTINCT(product_tmpl_id))
+        FROM product_template_product_tag_rel
```

Comment:
Would you consider indenting/prettifying the query?

I consider that, when dealing with queries, readability is more relevant than ever.

Reply:
What did you mean?